Jenkinsfile

```diff
@@ -13,7 +13,7 @@ node('puppet') {
     List mavenEnv = [
         "JAVA_HOME=${tool 'jdk8'}",
         "PATH+MVN=${tool 'mvn'}/bin",
-        'PATH+JDK=${JAVA_HOME}/bin',
+        "PATH+JDK=${tool 'jdk8'}/bin",
```

Comment:
@jglick I cannot identify when this broke in the history of this repo -- is it possible that this is a Pipeline regression? From previous builds it seemed to not pick up the expected java from the JAVA_HOME based PATH. Note this does not concat PATH, but uses PATH+WHATEVER.